### ISSUE TITLE
Segmentation fault in cld-3.4.3

### ISSUE THREAD

**Issue by coezbek** [akihikodaki/cld3-ruby#35]
I am running on Ubuntu 20.04 with Ruby 2.7.2 / cld3-ruby 3.4.3 on WSL and get an segmentation fault, wenn (by accident) I passed identical values to min and max to the constructor `CLD3::NNetLanguageIdentifier.new(min, max)`.

```
/home/coezbek/.rvm/gems/ruby-2.7.2/gems/cld3-3.4.3/lib/cld3.rb:79: [BUG] Illegal instruction at 0x00007f6501ff37a0
ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-linux]

-- Control frame information -----------------------------------------------
c:0040 p:---- s:0231 e:000230 CFUNC  :new_NNetLanguageIdentifier

libprotobuf.so.17.0.0
```

**Comment by akihikodaki** [comment]
Fixed with commit 585e178d3b9a9634d961ecd3472c193265731e95.

**Comment by akihikodaki** [comment]
It no longer traps and raises `ArgumentError` gracefully, but I doubt that it should reject such arguments at first place. I opened google/cld3#65 so let's see if the upstream maintainers agree.
